Engine/Formatter.cs

```diff
@@ -45,6 +45,7 @@ public class Formatter
                 "PSUseCorrectCasing",
                 "PSAvoidUsingCmdletAliases",
                 "PSAvoidUsingDoubleQuotesForConstantString",
+                "PSAvoidSemicolonsAsLineTerminators"
```

Comment:
Add trailing comma so that future line commits do not need to touch this line
```suggestion
                "PSAvoidSemicolonsAsLineTerminators",
```